packages/notifi-react-card/lib/components/subscription/subscription-card-views/PreviewCard.tsx

```diff
@@ -28,7 +29,9 @@ export type PreviewCardProps = Readonly<{
 export const PreviewCard: React.FC<PreviewCardProps> = ({
   classNames,
   data,
+  hideBackClick,
   inputDisabled,
+  hideEditButton,
```

Comment:
similar to the input icon, i think we can achieve this with CSS overrides rather than props

Reply:
let me try that